tests/rustdoc/issue-96381.rs

```diff
@@ -8,7 +6,6 @@ trait Foo<T>: Sized {
 
 impl Foo<usize> for () {
     fn bar(i: _, t: _, s: _) -> _ {
-        //~^ ERROR the placeholder `_` is not allowed within types on item signatures for functions
```

Comment:
This test is not being run (and I guess hasn't been since it was first added). It needs to go in rustdoc-ui, not rustdoc/.